src/engine/objects/SimBox.ts

```diff
@@ -78,4 +76,12 @@ export class SimBox extends SimObject {
   setBaseColor(color: number): void {
     (<THREE.MeshStandardMaterial>this._mesh.material).color.set(color);
   }
+
+  getBodySpecs(): {} {
```

Comment:
This could be a readonly getter

`get bodySpecs(): BodyDef { return this._bodySpecs; }`